Code/Editor/EditorFramework/Assets/Implementation/AssetDocument.cpp

```diff
@@ -98,7 +98,7 @@ void ezAssetDocument::InternalAfterSaveDocument()
     if (flags.IsAnySet(ezAssetDocumentFlags::AutoTransformOnSave))
     {
       /// \todo Should only be done for platform agnostic assets
-      auto ret = ezAssetCurator::GetSingleton()->TransformAsset(GetGuid(), false);
+      auto ret = ezAssetCurator::GetSingleton()->TransformAsset(GetGuid(), ezTransformFlags::None);
```

Comment:
Since saving is always a "manual" thing, one could probably argue to pass the manual flag in here. Would make sense, as an asset that is "only transform manually" but also "transform on save" might need this. Though I guess we hacked that part to work right elsewhere, since it already works with Prefabs, but this change may be cleaner still.

Reply:
Done.